### ISSUE TITLE
Any plans on supporting safari?

### ISSUE THREAD

**Issue by augustocdias** [huhu/rust-search-extension#87]
Do you plan to support safari or [alfredapp](https://www.alfredapp.com)?

**Comment by Folyd** [comment]
Hi, @augustocdias. I'd love to support Safari, but according to MDN's web extension [compatibility chart](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Browser_support_for_JavaScript_APIs#omnibox): Safari doesn't support **omnibox** API, which is essential to this extension. 

I'd love to seek support from the community for the Alfred version since I'm not sure I have enough time and energy right now.

**Comment by Kloenk** [comment]
Yes, omnibox seems to be missing. I tried to convert it to a safari extension, but it does not work, and only shows a warning about omnibox.

**Comment by AurevoirXavier** [comment]
Up.

Looking forward about the alfred version.

**Comment by Folyd** [comment]
I built another brand-new product called https://query.rs, welcome to give it a try. :)

> Query.rs was inspired by Rust Search Extension. Due to the partial support of the [omnibox API](https://developer.chrome.com/docs/extensions/reference/api/omnibox) by some browsers, Rust Search Extension is not available on browsers such as [Safari](https://github.com/huhu/rust-search-extension/issues/87), [Vivaldi](https://github.com/huhu/rust-search-extension/issues/10), [Arc](https://github.com/huhu/rust-search-extension/issues/230), etc. Query.rs addresses this gap.
